pyproject.toml

```diff
@@ -11,7 +11,7 @@ authors = [
 ]
 description = "A language-agnostic LSP client in Python, with a library interface. Intended to be used to build applications around language servers. Currently multilspy supports language servers for Python, Rust, Java, Go, JavaScript and C#. Originally appeared as part of Monitor-Guided Decoding (https://github.com/microsoft/monitors4codegen)"
 readme = "README.md"
-requires-python = ">=3.7"
+requires-python = ">=3.8 <4.0"
```

Comment:
Why is this required?

Reply:
if you change it back and run `poetry lock`
it will complain that requests (2.32.3) requires Python >=3.8

Reply:
According to the docs you need `>=3.8,<4` with the `,`.
https://python-poetry.org/docs/pyproject/#requires-python

I haven't tested with poetry, but otherwise at least with `uv` this causes an issue:

```
$ uv add git+https://github.com/AviAvni/multilspy.git --rev 3e7e4684f1eb43831f3b092817aa6a7701e175ab
 Updated https://github.com/AviAvni/multilspy.git (3e7e468)
error: Failed to extract static metadata from `pyproject.toml`
  Caused by: Failed to parse version: Trailing `<4.0` is not allowed:
>=3.8 <4.0
^^^^^^^^^^
```

Reply:
poetry is ok

Reply:
@AviAvni, can you address this comment? We should not add "<4.0"